doc/install/contributing.rst

```diff
@@ -886,13 +886,17 @@ all the documentation and additionally execute just your example or tutorial
 you expect).
 
 .. note::
-   On Windows, to use the pattern approach, use the following two lines:
+   If you are using a *Windows command shell*, to use the pattern approach,
+   use the following two lines:
 
    .. code-block:: python
 
-      set PATTERN={<REGEX_TO_SELECT_MY_TUTORIAL>}
+      set PATTERN=<REGEX_TO_SELECT_MY_TUTORIAL>
```

Comment:
are we sure the curly brackets also should be dropped in this case? Something we could not test when we came across our problem!

Reply:
I'm pretty sure they were in there as an attempt at formatting; I think `{}` will italicize parts of code spans in rST, but maybe only for inline code spans (not code blocks)?

Reply:
Ohhh. Hadn't even considered that. 😅 
Well, otherwise LGTM!